templates/components/llamaindex/typescript/streaming/annotations.ts

```diff
@@ -65,6 +62,36 @@ export function retrieveMessageContent(messages: Message[]): MessageContent {
   ];
 }
 
+function getFileContent(file: DocumentFile): string {
+  const fileMetadata = file.metadata;
+  let defaultContent = `=====File: ${fileMetadata.name}=====\n`;
+  // Include file URL if it's available
+  const urlPrefix = process.env.FILESERVER_URL_PREFIX;
+  let urlContent = "";
+  if (urlPrefix) {
+    if (fileMetadata.url) {
+      urlContent = `File URL: ${fileMetadata.url}\n`;
+    } else {
+      urlContent = `File URL (instruction: do not update this file URL yourself): ${urlPrefix}/output/uploaded/${fileMetadata.name}\n`;
+    }
+  } else {
+    console.warn(
+      "Warning: FILESERVER_URL_PREFIX not set in environment variables. Can't use file server",
+    );
+  }
+  defaultContent += urlContent;
+
+  // Include document IDs if it's available
+  if (fileMetadata.refs) {
+    defaultContent += `Document IDs: ${fileMetadata.refs}\n`;
+  }
```

Comment:
for what do we need to send the ID of the documents stored in the vector store to the LLM?

Reply:
it's used for in-text citing

Reply:
can you add a comment